clients/python/client/osparc/_info.py

```diff
@@ -8,5 +8,5 @@ def openapi() -> Dict[str, Any]:
     Get openapi specification of automatically-generated part of this client
     """
     return json.loads(
-        (Path(__file__).parent.parent / "data" / "openapi.json").resolve().read_text()
+        (Path(__file__).parent / "data" / "openapi.json").resolve().read_text()
```

Comment:
Careful! Notice that when the package is installed,  the file `opeanpi.json` will not be in that path.
The way to do this is 
a) adding the file to the package manager (i.e. setup.py)
b) retrieve the path to the package using `pck_resources`
```
import pkg_resources

# Assuming your data file is located in the same directory as setup.py
openapi_path = pkg_resources.resource_filename(__name__, 'openapi.josn')
```

Reply:
I think this actually works. I have a test which runs that function from the installed wheel on the ci. So I think it should work. If OK for you I will leave it as is now to not spend too much time on this detail(?)